data/projectRouter.js

```diff
@@ -0,0 +1,92 @@
+const express = require( 'express' );
+const Projects = require( './helpers/projectModel' );
+const router = express.Router();
+
+//RECIEVING ALL PROJECTS
+router.get( '/' , async ( req, res ) => {
+    try {
+        const projects = await Projects.get( req.query.id );
```

Comment:
you dont need to pass anything here as there is no query object, you can just pass Projects.get() invoked with nothing